frontend/src/pages/cfp/index.tsx

```diff
@@ -57,18 +58,24 @@ const CfpSectionOrClosedMessage: React.SFC<{ open: boolean }> = ({ open }) => {
 
 export const CFPPage: React.SFC = () => {
   const [isLoggedIn, _] = useLoginState();
+  const language = useCurrentLanguage();
+
   const code = process.env.conferenceCode;
 
   const { loading, data } = useIsCfpOpenQuery({
     variables: { conference: code },
   });
-
   return (
     <Fragment>
       <FormattedMessage id="cfp.pageTitle">
         {(text) => <MetaTags title={text} />}
       </FormattedMessage>
-      <Introduction />
+      <Introduction
+        deadline={formatDeadlineDate(
+          data?.conference.cfpDeadline?.end,
```

Comment:
this doesn't seem correct, if we pass undefined `new Date()` would fail 🤔

Reply:
![image](https://user-images.githubusercontent.com/3382153/145659841-cf9b3185-5d34-449c-beea-90cf599cff8d.png)
this is what happens 🤔  we need to improve this CFP page after we merge my PR #2540